### ISSUE TITLE
dataset of clipart

### ISSUE THREAD

**Issue by ShifuShen** [harsh-99/SCL#1]
Hello,  when I  try to run the experiment of VOC to Clipart, the result can not match as the paper reported by strong-weakly alignment, so I want to know the result is based on all the test data? And the hyperparameter(such as learning rate) of this experiment is set as?

**Comment by harsh-99** [comment]
Hi, 
Even when we trained their model we achieved lesser performance as compared to what strong weak alignment paper have stated. As mentioned in the paper while training and testing we have used whole data set i.e. 1k images. The hyper parameter used is same as the default ones. However we are able to achieve much better performance by our method.

**Comment by harsh-99** [comment]
I guess you problem is solved so I am closing this issue.
